### PR TITLE
Zombienet: add disputes test

### PR DESCRIPTION
**Pull request by sandreim**

### CODE REVIEW

**Review by ordian — APPROVED** [review]
Does it include functionality of [0001 test](https://github.com/paritytech/polkadot/blob/master/node/malus/integrationtests/0001-dispute-valid-block.feature)? Should we replace it if so?

**Review by drahnr — APPROVED** [review]

**Comment by sandreim** [comment]
> Does it include functionality of [0001 test](https://github.com/paritytech/polkadot/blob/master/node/malus/integrationtests/0001-dispute-valid-block.feature)? Should we replace it if so?

Yes it does. That test is simpler and smaller scale focused on testing the malus functionality. I wouldn't remove it or change, as it can help reason about failures of this new test.

File: zombienet_tests/functional/0002-parachains-disputes.feature [hunk 1]
```diff
@@ -0,0 +1,69 @@
+Description: Disputes initiation, conclusion and lag
+Network: ./0002.parachains-disputes-local.toml
```
Comment (sandreim) [inline zombienet_tests/functional/0002-parachains-disputes.feature:2]:
```suggestion
Network: ./0002.parachains-disputes.toml
```

File: zombienet_tests/functional/0002-parachains-disputes.feature [hunk 2]
```diff
@@ -0,0 +1,69 @@
+Description: Disputes initiation, conclusion and lag
+Network: ./0002.parachains-disputes.toml
```
Comment (sandreim) [inline zombienet_tests/functional/0002-parachains-disputes.feature:2]:
```suggestion
Network: ./0002-parachains-disputes.toml
```

**Pull request merged**